modules/installation-openshift-local.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * getting_started/openshift-overview.adoc
+// * installing/index.adoc
+
+:_content-type: CONCEPT
+[id="installation-openshift-local_{context}"]
+= {product-title} Local Overview
+
+Red Hat {product-title} Local supports rapid application development to get started building {product-title} clusters. {product-title} Local is designed to run on a local computer to simplify setup and testing, and to emulate the cloud development environment locally with all of the tools needed to develop container-based applications. 
+
+Whatever your programming language, {product-title} Local hosts your application and brings a minimal, preconfigured Red Hat {product-title} 4.1 or newer cluster to your local PC without the need for a server-based infrastructure.
+
+On a hosted environment, {product-title} Local supports creating microservices, building microservices into images, and running them in Kubernetes-hosted containers right on your laptop or desktop running Linux, macOS, or Windows 10 and later. 
```

Comment:
Slight suggestion to reduce the redundancy of "running"

```suggestion
On a hosted environment, with {product-title} Local, you can create microservices, convert them into images, and run them in Kubernetes-hosted containers directly on your laptop or desktop running Linux, macOS, or Windows 10 or later.
```

Feel to ignore/revise another way!